sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokersTestHelper.java

```diff
@@ -15,14 +15,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.sdk.transforms.reflect.testhelper;
+package org.apache.beam.sdk.transforms.reflect;
```

Comment:
We can't do this - actually what we are testing here is specifically that `DoFnInvokers` works when the `DoFn` is in a different package. We will have to find another way to rewrite the test.

Perhaps instead of trying to force Mockito to work we should just add some boilerplate to the different `DoFn` classes to get the same tests. It should really be very simple, just tracking what is called.